demisto_sdk/commands/common/git_util.py

```diff
@@ -55,6 +55,8 @@ def __init__(
 
         if isinstance(path, str):
             repo_path = Path(path)
+        elif isinstance(path, self.REPO_CLS):
```

Comment:
Update the type declaration in line 52 to allow `Repo` as well as the existing `str, Path`.

Reply:
done